src/new/wallet.js

```diff
@@ -1426,7 +1426,7 @@ class HathorWallet extends EventEmitter {
    *    'skipSpent': if should not include spent utxos (default true)
    *  }
    *
-   * @return {Array} Array of objects with {tx_id, index, address, authorities} of the authority output. Returns null in case there are no utxos for this type
+   * @return {Array|null} Array of objects with {tx_id, index, address, authorities} of the authority output. Returns null in case there are no utxos for this type
    **/
   selectAuthorityUtxo(tokenUid, filterUTXOs, options = {}) {
```

Comment:
There's as place calling this function this way:

```js
    const destroyInputs = this.selectAuthorityUtxo(tokenUid, filterUtxos, { many: true });

    if (destroyInputs.length < count) {
      return Promise.reject({ success: false, message: ErrorMessages.NO_UTXOS_AVAILABLE, errorData: { requestedQuantity: count, availableQuantity: destroyInputs.length } });
    }
```

Shouldn't we treat there the case when it returns `null`?

Reply:
This is a pre-existing problem, so if this is really a problem, and you think it should be made in another PR, I'm ok.

Reply:
The `many: true` option will always return an array, an empty array in case none are found.